integration-tests/database/index.ts

```diff
@@ -61,7 +64,10 @@ switch (testScenario) {
         // Dynamodb server side encryption disabled.
         dynamodbArgs = {
             hashKey: "test",
-            attributes: [],
+            attributes: [{
```

Comment:
this was a change made to dynamodb.Table since the 2.0 aws package that this originally referenced